Second pull request, added page content

ReshyResh opened the pull request:
# Second pull request changes 

 - Added grid layout for projects section.
 - Flex-based positioning of the items inside the grid.
 - Added About page, with responsive button and images.
 - Informative squares with fitted background images.
 - Input form.
 - Social icons added using font-awesome.

 
# Notes
- Site is heavily optimized for mobile versions, desktop might look a bit off.
- For some reason GitHub Pages wouldn't display the page using the css file if it is in another folder, so I had to move it to the root folder.

All steps were done using GitHub flow, commits are descriptive and all the guidelines from the tutorials were followed.
The color palette might look slightly different from the template I used (Template n.5), but it's a personal choice and it doesn't look bad in dark mode.

# [Live Demo](https://reshyresh.github.io/Portfolio/)

mimipeshy commented on index.html:
```diff
@@ -3,23 +3,155 @@
+    <div class="shape-square"></div>
+    <section class="main-container">
+      <h1 class="title">Get Started</h1>
+      <p class="presentation">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna</p>
+      <input type="text" class="input-form" placeholder="Your name">
+      <input type="text" class="input-form" placeholder="Your email">
+      <input type="text" class="text-form" placeholder="Enter your message here..">
+      <button class="button-main" type="submit"><span class="button-text">Start Collaboration</span></button>
```
- [ ] For form elements, let's use the `form` tag instead, this tells the browser that its a form
- [ ] In the UI design, form elements and the button overlap the paragraph, kindly work on this as well

mimipeshy reviewed (changes requested):
## Hi @ReshyResh 👋🏾 👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
- Great git workflow ✔️ 
- Great PR summary ✔️
- Good UI design looks almost similar to the design you chose ✔️  

## Required Changes ♻️

_Check comments under the review_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
You can consider 👍🏾 

- Making your portfolio responsive across all mobile devices, currently, it's only responsive to devices with 428px to 528px, it would be nice to consider smaller devices as well.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
------

ReshyResh commented:
# Hi @mimipeshy 
Thanks for your suggestions, I managed to implement every single one in the updated code.

- Implemented use of lists in the projects section, positioning them correctly.
- Used hx tag for project titles instead of span.
- Created new section for a small description for the project using paragraph tag.
- Grouped up the programming language boxes in a list and positioned them correctly.
- Added a form tag for the user input.
- Fixed overlap issue.

# [Live Demo](https://reshyresh.github.io/Portfolio/)


Cheers and Happy coding!👏

carlos-ssh commented on style.css:
```diff
@@ -0,0 +1,344 @@
+  background-image: url("img/shapes.png");
+}
+
+.grid-projects {
+  display: grid;
+  left: 15vw;
+  right: 15vw;
+  width: 70vw;
+  height: 100%;
+  position: relative;
+  background: #fff;
+}
```
- Your items are floating to the left and should be at the center of the page. You could fixed using: a grid-template-area in example.

carlos-ssh commented on style.css:
```diff
@@ -0,0 +1,344 @@
+  padding-left: 10px;
+}
+
+.about-section {
+  left: 15vw;
+  right: 15vw;
+  width: 70vw;
+  height: 100vh;
+  position: relative;
+}
```
- Try to center at the page the items that contain this section.

carlos-ssh commented on style.css:
```diff
@@ -0,0 +1,344 @@
+  z-index: -1;
+}
+
+.main-container {
+  display: flex;
+  left: 15vw;
+  right: 15vw;
+  width: 70vw;
+  position: relative;
+  flex-direction: column;
+}
```
- Should have items centered.

carlos-ssh commented on README.md:
```diff
@@ -16,7 +16,8 @@ Beginning of the personal portfolio, starting with the mobile version.
 
 - Interactive main button
  
-  ![9hO5zfC](https://user-images.githubusercontent.com/85108160/124786724-3d0f8a00-df48-11eb-90cd-d853d4c7415f.gif)
+  ![v0MWP4s](https://user-images.githubusercontent.com/85108160/124942160-ac9a7d80-e00b-11eb-91c5-daaf14b4e193.gif)
+
 
 ## Authors
 
```
- Should be good to have a link for the design that you selected on figma.

carlos-ssh reviewed (changes requested):
Hi @ReshyResh , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

carlos-ssh reviewed (approved):
Hi @ReshyResh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
